language-support/java/codegen/src/ledger-tests/daml/Wolpertinger.daml

```diff
@@ -51,21 +50,3 @@ template Wolpertinger
                     timeOfBirth = tob
                     parents = [self, mateId]
                     colors = dedup (this.colors ++ mate.colors)
-
-
-primitiveTest =
```

Comment:
this was never called - I ran the tests that depend on this file after adding an `error` in this scenario and they all succeeded anyway